### ISSUE TITLE
Migrate `:eq(n)` jQuery selectors

### ISSUE THREAD

**Issue by simonihmig** [simonihmig/ember-native-dom-helpers-codemod#8]
See https://github.com/simonihmig/ember-native-dom-helpers-codemod/pull/7#discussion_r117208937

**Comment by knownasilya** [comment]
Got an error on this, also using global helpers it seems also errors:

```zsh
 ▲ ~/sandbox/ember-twiddle ember-native-dom-helpers-codemod --type=acceptance tests/acceptance                             left-panel :: 10d :: ⬡
Processing 20 files... 
Spawning 7 workers...
Sending 3 files to free worker...
Sending 3 files to free worker...
Sending 3 files to free worker...
Sending 3 files to free worker...
Sending 3 files to free worker...
Sending 3 files to free worker...
Sending 2 files to free worker...
 ERR tests/acceptance/dependency-test.js Transformation error
SyntaxError: Unexpected token (86:8)
    at Parser.pp.raise (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/location.js:24:13)
    at Parser.pp.unexpected (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/util.js:82:8)
    at Parser.pp.semicolon (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/util.js:69:81)
    at Parser.pp.parseExpressionStatement (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/statement.js:427:8)
    at Parser.parseExpressionStatement (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/plugins/flow.js:676:20)
    at Parser.pp.parseStatement (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/statement.js:142:21)
    at Parser.parseStatement (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/plugins/flow.js:655:22)
    at Parser.pp.parseBlock (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/statement.js:442:21)
    at Parser.pp.parseFunctionBody (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:747:22)
    at Parser.parseFunctionBody (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/plugins/flow.js:643:20)
 ERR tests/acceptance/twiddles-test.js Transformation error
SyntaxError: Unexpected token (55:8)
    at Parser.pp.raise (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/location.js:24:13)
    at Parser.pp.unexpected (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/util.js:82:8)
    at Parser.pp.semicolon (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/util.js:69:81)
    at Parser.pp.parseExpressionStatement (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/statement.js:427:8)
    at Parser.parseExpressionStatement (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/plugins/flow.js:676:20)
    at Parser.pp.parseStatement (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/statement.js:142:21)
    at Parser.parseStatement (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/plugins/flow.js:655:22)
    at Parser.pp.parseBlock (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/statement.js:442:21)
    at Parser.pp.parseFunctionBody (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/parser/expression.js:747:22)
    at Parser.parseFunctionBody (/usr/local/lib/node_modules/ember-native-dom-helpers-codemod/node_modules/babel-core/node_modules/babylon/lib/plugins/flow.js:643:20)
All done. 
Results: 
4 errors
8 unmodified
0 skipped
8 ok
Time elapsed: 4.199seconds 
```

Tests here https://github.com/knownasilya/ember-twiddle/tree/left-panel/tests

**Comment by simonihmig** [comment]
@knownasilya Thanks for reporting this! Could you please create a separate issue for this, I *think* it is not related. It seems there is a parsing error even before the codemod actually runs...

**Comment by simonihmig** [comment]
Closed by #23
